src/component/greeting/Greeting.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import './greeting.scss';
+
+const greet = (name) => {
+  let time = new Date().getHours();
+  let newName = name ? `, ${name}` : '';
+
+  if (time < 3 && time >= 22) {
+    return `“Good night${newName}!”`;
```

Comment:
I think you don't need this quotes '"' here, text will interpolate anyway because of ``

Reply:
it's just decorating